### DISCUSSION TITLE
Cannot package UNO application with "Package and Publish > Create App Packages"

### DISCUSSION THREAD

**Discussion by Looky1173** [unoplatform/uno#14464] [Q&A]
Hello,

I've generated a UNO application with the Template Wizard, but I am unable to publish it (the Windows version). When using _Package and Publish > Create App Packages_, I get the error _No valid build configurations were found. If you are building packages for the Store, make sure you have at least one non-Debug configuration for the project._ Yet I do have a valid release configuration for the project...

![image](https://github.com/unoplatform/uno/assets/60521810/c233d144-68af-403f-a8e5-9cbe5bc90c92)

**Comment by jeromelaban** [comment]
Thanks for the report! 

@nickrandolph does this look familiar?
